packages/atomic/cypress/e2e/smart-snippet.cypress.ts

```diff
@@ -114,8 +107,13 @@ describe('Smart Snippet Test Suites', () => {
         .init();
     });
 
-    CommonAssertions.assertConsoleError(true);
-    CommonAssertions.assertContainsComponentError(SmartSnippetSelectors, true);
+    it('should display errors', () => {
+      CommonAssertions.assertConsoleErrorWithoutIt(true);
```

Comment:
Don't see why these need to be within an `it` and have the assertions be without the `it`. I don't see anything about this pattern in the best practices.

Reply:
Same comment goes for a bunch of different places in this file hehe